lib/index.js

```diff
@@ -55,11 +55,10 @@ class Rooftop {
       if (transformFn === true) transformFn = transform
       else if (transformFn === false) transformFn = (x) => x
 
-      return this.client[name].get(options)
+      return W.resolve(this.client[name].get(options))
         .then((p) => W.map(p, transformFn))
-        .then((v) => m[name] = v)
-        .then(() => m)
-
+        .then((v) => { m[name] = v })
```

Comment:
It's not necessary for the code to work (obviously), it's more of a convention/habit. If you are making a transformation but *not* returning the value down the chain, you use `tap`, which doesn't return the value. If you are intending to change the value you are passing down the chain, you use `then` which does.